docs/how-to-guides/renkulab/ssh-into-sessions.rst

```diff
@@ -0,0 +1,256 @@
+    command under your project's Settings tab. Or, form it yourself in the style
+    of the following URL: 
+
+    .. code-block:: console
+    
+        $ renku clone https://renkulab.io/gitlab/user/my-project.git
+
+.. note::
+
+    **Do you want to use password-protected ssh keys for your RenkuLab
+    sessions?** By default, the first time you launch an SSH session, Renku will
+    create a password-less key pair for use in authenticating with RenkuLab
+    sessions. If you would prefer to have a password, you can do so by running
+    (before launching a session) ``renku session ssh-setup -k
+    <path_to_keypair>`` and providing a path to a password protected key of your
+    choice. If you've already launched a session and would like to switch to a
+    different key pair, also include the ``--force`` option.
+
+Launch an SSH-enabled Session
+-----------------------------
+
+#.  Navigate so your current working directory is the project you want to start
+    a session in:
+
+    .. code-block:: console
+    
+        $ cd my-project
+
+
+#.  Log in to RenkuLab with :meth:`renku login <renku.ui.cli.login>`: 
+
+    .. code-block:: console
+
+        $ renku login renkulab.io
+
+
+#.  Pull any changes. This makes sure that the session you start is for the
+    latest version of your project.
+
+    .. code-block:: console
+
+        $ git pull
+
+
+#.  Start a session with :meth:`renku session start <renku.ui.cli.session>`,
+    using the ``--ssh`` flag and ``-p renkulab`` to specify running the session
+    remotely on RenkuLab:
+
+    .. code-block:: console
+
+        $ renku session start -p renkulab --ssh Your system is not set up for
+        SSH connections to Renku. Would you like to set it up? [y/N]: y [...]
+        Session user-myproject-02a9e407 successfully started, use 'renku session
+        open --ssh user-myproject-02a9e407' or 'ssh
+        renkulab.io-myproject-02a9e407' to connect to it
+    
+    .. note::
+
+        **Curious what's happening under the hood?** This command starts a new
+        session on RenkuLab. But first, it adds your local SSH keys to the
+        ``allowed_keys`` in the project and pushes those changes to RenkuLab. If
+        this is your first time using the SSH feature on RenkuLab, Renku creates
+        an SSH key pair for you. Once the session is started, Renku creates an
+        SSH connection entry in your local SSH config for that session id. This
+        SSH config entry can be used with your SSH client or tools like VSCode.
+
+
+Open an SSH Session via a Shell
+-------------------------------
+
+You can use :meth:`renku session open <renku.ui.cli.session>` to open an SSH
+connection directly. This will open a terminal in your RenkuLab Project session
+environment.
+
+.. code-block:: console
+
+    $ renku session open --ssh <session-id>
+    venv ▶ ~ ▶ $
+
+(If prompted to accept host keys, confirm with ``yes``).
+
+To exit the SSH shell, simply type ``exit``.
```

Comment:
eh, but the keyboard shortcut is different on different operating systems... I think current state is sufficient, since our target audience for this feature is people who already know how to use SSH.